cargo-concordium/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
+
+
 ## Unreleased changes
+
+- Support schema types for LEB128 and byte arrays.
+- Support schema modules which includes version information.
```

Comment:
Change schema type for `[u8]`

Reply:
Whether `[u8]` use the new type will depend on the version of `concordium-std`. 
`cargo-concordium` just needs to support the new schema types.